OutboundCallReminder/OutboundFunction_AzurePortal/SendNotification/run.csx

```diff
@@ -0,0 +1,89 @@
+#r "Newtonsoft.Json"
+
+#load "Logger.csx"
+#load "Phonecall.csx"
+#load "Sendsms.csx"
+
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+
+public static async Task<IActionResult> Run(HttpRequest req, ILogger log)
```

Comment:
Add try catch block for entire block and return the failure response in case of exceptions.